status-im/status-react
# Custom Option Menu for TX Parameters on SEND Flow

fixes #21020


### Summary
Implemented menu for customizing transaction parameters on send flow.
[Figma link](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-201810&m=dev)


https://github.com/user-attachments/assets/46979639-8ac0-4b91-b21e-6b195eb7ced5



### Review notes
This PR adds feature flag, two menus and adjusts `settings_item` quo component to accept emoji as settings menu item.
There is a hardcoded dummy data inside because so far we don't have access to any real data. This is a subject for another issues in [epic](https://github.com/status-im/status-mobile/issues/21016).

#### Platforms
- Android
- iOS


#### Areas that maybe impacted


##### Functional

- wallet / transactions

### Steps to test


- Open Status
- Navigate to settings, feature flags and enable `::wallet.transaction-params` flag
- Start send flow and navigate to transaction confirmation screen
- click the transaction settings button to see the bottom sheet with menu



status: ready
